examples/ecs-fargate-service/.terraform.tfstate.lock.info

```diff
@@ -0,0 +1 @@
+{"ID":"8389cea4-8f76-b412-193a-7957e928771c","Operation":"OperationTypeApply","Info":"","Who":"brikis98@Yevgeniys-MacBook-Air.local","Version":"1.6.2","Created":"2024-03-07T18:41:05.861475Z","Path":"terraform.tfstate"}
```

Comment:
Is this file included intentionally?

Reply:
Whoops, nope, good catch. Fixed: https://github.com/gruntwork-io/terragrunt-infrastructure-modules-example/pull/44/commits/3bb3ee2e8275f3fcf75d372cd633e5a5b10e59b5 and https://github.com/gruntwork-io/terragrunt-infrastructure-modules-example/pull/44/commits/36ed1c79c24db1390b2f535fb55a18db87ad5960.